adapters/displayio/displayiotest/exemplary/multi-format.json

```diff
@@ -0,0 +1,280 @@
+{
+  "mockBidRequest": {
+    "id": "requestId10111011101110111011",
+    "app": {
+      "bundle": "displayio101",
+      "cat": [
+        "IAB1"
+      ],
+      "id": "1011",
+      "name": "Display.io 101",
+      "pagecat": [
+        "IAB1"
+      ],
+      "publisher": {
+        "id": "1101"
+      },
+      "storeurl": "https://apps.apple.com/app/id1000000000"
+    },
+    "at": 2,
+    "cur": [
+      "USD"
+    ],
+    "imp": [
+      {
+        "id": "impId10111011101110111011",
+        "tagid": "1011",
+        "secure": 1,
+        "ext": {
+          "bidder": {
+            "placementId": "1011",
+            "publisherId": "101",
+            "inventoryId": "1011"
+          }
+        },
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            },
+            {
+              "w": 320,
+              "h": 50
+            }
+          ]
+        },
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ],
+          "protocols": [
+            2,
+            5
+          ],
+          "w": 640,
+          "h": 480
+        },
+        "bidfloor": 0.5,
+        "bidfloorcur": "USD"
+      }
+    ],
+    "regs": {
+      "ext": {
+        "gdpr": 0
+      }
+    },
+    "source": {},
+    "tmax": 1000,
+    "user": {
+      "id": "user10111011101110111011"
+    },
+    "ext": {
+      "displayio": {
+        "placementId": "1011",
+        "inventoryId": "1011"
+      }
+    }
```

Comment:
>Is it typical for this information to exist in the request.ext of the incoming request?
No, no matter what will be send under `request.ext.displayio`, it will be replaced by adapter code.
I dropped it from incoming request tests to keep it straight